renderdoc/replay/entry_points.cpp

```diff
@@ -573,7 +573,23 @@ void extractDeviceIDAndIndex(const string &hostname, int &index, string &deviceI
 
   deviceID = c;
 }
-string adbExecCommand(const string &device, const string &args)
+Process::ProcessResult execCommand(const string &cmd, const string &workDir = ".")
+{
+  RDCLOG("COMMAND: %s", cmd.c_str());
+
+  size_t firstSpace = cmd.find(" ");
+  string exe = cmd.substr(0, firstSpace);
+  string args = cmd.substr(firstSpace + 1, cmd.length());
+
+  Process::ProcessResult result;
+  Process::LaunchProcess(exe.c_str(), workDir.c_str(), args.c_str(), &result);
+  if(result.strStdout.length())
+    RDCLOG("STDOUT:\n%s", result.strStdout.c_str());
+  if(result.strStderror.length())
+    RDCLOG("STDERR:\n%s", result.strStderror.c_str());
```

Comment:
Okay.  This was useful during development, but probably doesn't need to be so verbose.  In the future I think it would be good to return the exit code of commands, but I didn't see a good way to do that right now.